README.md

```diff
@@ -1 +1,18 @@
-# js-restaurant-page
\ No newline at end of file
+# JS Restaurant Page
+# SOLO Project:
+## Project Specifications:
+
+This project comes after the introduction of the ES6 class keyword, ES6 modules, and webpack. The main goal of this project is for the student to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using webpack.
+
+You can find the original project specification [here](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)
+
+## Live Version
+
+[here](https://raw.githack.com/men32z/js-restaurant-page/the-page/dist/index.html)
```

Comment:
[Optional]

Remember to update this address before merging. It is currently pointing to the `the-page` branch. The page gets broken if the branch is deleted